src/infrastructure/dataSchemes/allDataScheme.ts

```diff
@@ -0,0 +1,10 @@
+import { configDataScheme } from './configDataScheme'
+import { pixelDataScheme } from './pixelDataScheme'
+import { PlotDataScheme } from './plotDataScheme'
+
+export interface allDataScheme {
+  appVersion: string
```

Comment:
Not sure if we need app version in the json data, but currently each scheme has its own scheme version and all-data scheme has an app version.

Another option is to have only one global scheme version on the allDataScheme, and not to have app version or have both

Reply:
If it will be a single JSON, then having just one would be fine. However, if multiple JSONs are generated upon output, it might be better to have each separately. I assume it's the former, though

Reply:
ah, this message looks outdated 😓

Reply:
AllDataSchema will be one single JSON, but each of PlotDataSchema and ConfigDataSchema might be imported in other export formats.
So for now I think it would be better to have schema versions for each🤔